packages/react/src/hooks/useCSSTransition.ts

```diff
@@ -0,0 +1,197 @@
+import type {
+  FloatingContext,
+  Placement,
+  ReferenceType,
+  Side,
+} from '@floating-ui/react';
```

Comment:
This import should be from relative file, no?
```suggestion
} from './types';
```